File: 2.0/java/numbers/getNumberLeaseConfig.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.numbers.model.NumberConfig;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        NumberConfig config = client.numberLeasesApi().getConfig("16502572135");
    }
}

File: 2.0/csharp/numbers/getNumberLeaseConfig.cs
using CallfireApiClient;
using CallfireApiClient.Api.Numbers.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        NumberConfig config = client.NumberLeasesApi.GetConfig("12132000384", "number,configType");
    }
}
